SystemRuntime.UnitTests/InMemoryCacheTests.cs

```diff
@@ -27,86 +27,6 @@ public void TestPolicyDeserialization()
             Assert.AreEqual(false, policy.DoThrowExceptionOnTimeout);
         }
 
-        [TestMethod]
-        public void TestInMemoryCacheStruct()

```

Comment:
Where were these tests moved to? - Searching using "CacheStruct", "CacheNull" finds no new tests.

Reply:
moved to: InMemoryCache_CacheInterfaceTests.cs

used the technique described in: 
http://softwareonastring.com/502/testing-every-implementer-of-an-interface-with-the-same-tests-using-mstest
 
this way I was able to verify ICache, IScopedCache and LayeredCache expected common behaviors on all derived implementations while **still** verifying cache type-specific nuances:
Get_NestedGetWithoutLock_2Hits
TestInMemoryCacheObjectMutated
TestInMemorySerializedCacheObjectMutated
...